pycytominer/cyto_utils/util.py

```diff
@@ -327,8 +327,15 @@ def get_pairwise_correlation(population_df, method="pearson"):
     # Check that the input method is supported
     method = check_correlation_method(method)
 
-    # Get a symmetrical correlation matrix
-    data_cor_df = population_df.corr(method=method)
+    # Get a symmetrical correlation matrix. Use numpy for non NaN matrices.
+    has_nan = np.any(np.isnan(population_df.values))
+    if method == "pearson" and not has_nan:
+        print('USING np.corr')
+        pop_names = population_df.columns
+        data_cor_df = np.corrcoef(population_df.T)
```

Comment:
```suggestion
        data_cor_df = np.corrcoef(population_df.transpose())
```

This makes reading a bit easier, and is the same function :)